gamemodes/terrortown/gamemode/shared/sh_item_module.lua

```diff
@@ -17,6 +17,15 @@ for i = 1, #itemsFiles do
 
 	local cls = string.sub(fl, 0, #fl - 4)
 
+	ITEM.CanBuy = ITEM.CanBuy or {}
```

Comment:
Please add a comment, because without knowing why you doing this, it seems weird :D

Reply:
Actually, your comment made me realize that it isn't even necessary, since the automatic cleanup always happens later on anyways. I removed it